react-app/src/components/NavBar.js

```diff
@@ -112,11 +119,16 @@ const StyledNavElement = styled.header`
     gap: 8px;
   }
 
+<<<<<<< HEAD
```

Comment:
Missed some merge markers when doing a merge. Looks like it was a bit of styling you had done to the navbar but that should all be resolved by #54 and should be able to just remove any of the changes you made.

Reply:
This was marked as resolved but it seems this one is still unresolved